src/components/StudentGroup.tsx

```diff
@@ -5,7 +5,7 @@ import styled from '@emotion/styled'
 import 'firebase/firestore'
 import { useFirestore, useFirestoreCollectionData, useUser } from 'reactfire'
 
-const ClassroomDiv = styled.div`
+const StudentGroupDiv = styled.div`
```

Comment:
we don't really need to know this is a `div` here.  Instead, take the opportunity to describe what this element *does*.  Is it a `StudentGroupContainer`?